### ISSUE TITLE
Pull request on naming of quick-copy options welcome?

### ISSUE THREAD

**Issue by bjohas** [willsALMANJ/Zutilo#121]
Hello,

it would be nice to be able to name the quickcopy_alt_1/2 entries (as well as the main entry) in the Zutilo menu. 

Would you be interested in a PR that achieves that?

**Comment by willsALMANJ** [comment]
Yes, that sounds like a good idea.

I think there are a few ways this could be achieved:

1. Add more about:config options for the labels that go with the about:config entries for the alternative quickcopy translators.
2. Look up the name of the selected quickcopy translator and include it in the label automatically.
3. Create a new UI in the preferences window to set the labels to be shown with the alternative quick copy commands.

Any of those three would be fine. It sounds like you are leaning towards the third option? The one thing about that is that if there is going to be a UI for setting the label it should be alongside UI choosing the translator (rather than setting the translator in about:config like is done now).

**Comment by willsALMANJ** [comment]
The exchange in #129 makes this seem like an even better idea. It seems like Zutilo has been getting more popular in the last couple years (probably because Zotero has been getting more popular), and the frequency of feature requests has gone up. I don't mind adding new things that are not overly complex, but I am a little concerned that as the number of features goes up the chances of things breaking with new versions of Zotero goes up as well. One way to help with that would be to use translators more. If it were easy to add new features as new translators, that might make it easier for outside contributors to add a feature instead of just requesting it or to fix a feature if it broke. I am not sure exactly how to use translators more, but it is something to think about.

**Comment by Guillawme** [comment]
> I am not sure exactly how to use translators more

From my very limited experience hacking on Zotero [1], at least any "QuickCopy" functionality one would want can be implemented with an export translator (that only exports the field one wants to "QuickCopy") and a keyboard shortcut binding (or context menu, or whatever is one's prefered user interface element for a copy-like action).

[1]: I only made an extremely simple export translator this afternoon (I will push it to GitHub soon). I had never written any JavaScript and had never hacked on Zotero before.

**Comment by willsALMANJ** [comment]
> at least any "QuickCopy" functionality one would want can be implemented with an export translator

Yes, at least all the functions that are copy item data to the clipboard could become wrappers around quickcopy and a custom translator. However, I think it would still be good to provide easy access to generally useful functions out of the box with Zutilo, so that once a generally useful translator was written other users could install and use it easily without having to separately install the translators they want and label them.

**Comment by redleafnew** [comment]
Change the label of alt 1 from UI is easier for user.
